app/services/download_attachments.rb

```diff
@@ -0,0 +1,63 @@
+class DownloadAttachments
+  SUBSCRIPTION = 'download_attachments'.freeze
+  TIMEOUT = 30
+  OPEN_TIMEOUT = 30
+  attr_reader :attachments, :target_dir, :token, :access_token
```

Comment:
would it be worth us taking the opportunity to rename `token` here to better describe what it is?

Reply:
Done!